examples/dask/hello_world/notebook.ipynb

```diff
@@ -0,0 +1,340 @@
+    "## 7. Install requirements.\n",
+    "# %pip install -r requirements.txt\n",
+    "# clear_output()  # optionally clear outputs\n",
+    "# To check your current working directory you can type `!pwd` in a cell and run it."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "metadata": {
+    "collapsed": false,
+    "jupyter": {
+     "outputs_hidden": false
+    },
+    "pycharm": {
+     "name": "#%%\n"
+    }
+   },
+   "outputs": [],
+   "source": [
+    "# Cell 2 - import modules to create part of the DAG from\n",
+    "# We use the autoreload extension that comes with ipython to automatically reload modules when\n",
+    "# the code in them changes.\n",
+    "\n",
+    "# import the jupyter extension\n",
+    "%load_ext autoreload\n",
+    "# set it to only reload the modules imported\n",
+    "%autoreload 1"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 3,
+   "metadata": {
+    "collapsed": false,
+    "jupyter": {
+     "outputs_hidden": false
+    },
+    "pycharm": {
+     "name": "#%%\n"
+    }
+   },
+   "outputs": [],
+   "source": [
+    "# Import modules\n",
+    "import pandas as pd\n",
+    "from dask import dataframe\n",
+    "from dask.distributed import (\n",
+    "    Client,\n",
+    "    LocalCluster,\n",
+    ")\n",
+    "\n",
+    "from hamilton import (\n",
+    "    ad_hoc_utils,\n",
+    "    driver,\n",
+    ")\n",
+    "from hamilton.plugins import h_dask"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 4,
+   "metadata": {
+    "collapsed": false,
+    "jupyter": {
+     "outputs_hidden": false
+    },
+    "pycharm": {
+     "name": "#%%\n"
+    }
+   },
+   "outputs": [],
+   "source": [
+    "# We'll place the data loaders into a new module\n",
+    "\n",
+    "def spend(spend_location: str, spend_partitions: int) -> dataframe.Series:\n",
+    "    \"\"\"Dummy function showing how to wire through loading data.\n",
+    "\n",
+    "    :param spend_location:\n",
+    "    :param spend_partitions: number of partitions to segment the data into\n",
+    "    :return:\n",
+    "    \"\"\"\n",
+    "    return dataframe.from_pandas(\n",
+    "        pd.Series([10, 10, 20, 40, 40, 50]), name=\"spend\", npartitions=spend_partitions\n",
+    "    )\n",
+    "\n",
+    "\n",
+    "def signups(signups_location: str, signups_partitions: int) -> dataframe.Series:\n",
+    "    \"\"\"Dummy function showing how to wire through loading data.\n",
+    "\n",
+    "    :param signups_location:\n",
+    "    :param signups_partitions: number of partitions to segment the data into\n",
+    "    :return:\n",
+    "    \"\"\"\n",
+    "    return dataframe.from_pandas(\n",
+    "        pd.Series([1, 10, 50, 100, 200, 400]), name=\"signups\", npartitions=signups_partitions\n",
+    "    )\n",
+    "\n",
+    "data_loaders = ad_hoc_utils.create_temporary_module(\n",
+    "    spend,\n",
+    "    signups,\n",
+    "    module_name=\"data_loaders\",\n",
+    ")"
```

Comment:
yeah I dont think the adhoc module works with dask? See other comment in the other PR.

Reply:
yep, you are right. i just read the doc string for the create temp module fn. thanks for pointing that out.